jina/hubble/hubio.py

```diff
@@ -541,7 +541,7 @@ def _get_prettyprint_usage(self, console, usage):
 
     @staticmethod
     @disk_cache_offline(cache_file=str(_cache_file))
-    def _fetch_meta(
+    def fetch_meta(
```

Comment:
why does this need to be public?

Reply:
we need to call it outside the class when getting the docker image for the hub executor